index.html

```diff
@@ -701,6 +701,11 @@
             </ol>
           </li>
         </ol>
+        <p>
+          An <dfn>orientation change source</dfn> is said to be from a
```

Comment:
What would the wording in the Fullscreen API be like? Paging @annevk to review the answer to that question :)

And won't `<code>gesture</code>` just make "gesture" monospaced as if it's the name of an event or similar?

Is it fair to say that the bits added here can *only* be used in a way that guarantees a bad frame?